easybuild/easyconfigs/a/ABINIT/ABINIT-8.0.8b-foss-2016b.eb

```diff
@@ -10,9 +10,9 @@ description = """ABINIT is a package whose main program allows one to find the t
 
 toolchain = {'name': 'foss', 'version': '2016b'}
 
-source_urls = ['http://ftp.abinit.org/']
+source_urls = ['https://www.abinit.org/sites/default/files/packages/']
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['abc9e303bfa7f9f43f95598f87d84d5d']
+checksums = ['37ad5f0f215d2a36e596383cb6e54de3313842a0390ce8d6b48a423d3ee25af2']
```

Comment:
note: MD5 checksum is unchanged with updated source URL, but I figured it made sense to update to SHA256 while touching these...